src/Shiny.BluetoothLE/Platforms/Android/Peripheral.cs

```diff
@@ -132,16 +134,30 @@ public override IObservable<IGattService> DiscoverServices()
                     .ListenForMe(BluetoothDevice.ActionBondStateChanged, this)
                     .Where(x => this.context.NativeDevice.BondState != Bond.Bonding)
                     .Subscribe(x => ob.Respond(this.PairingStatus == PairingState.Paired));
-
```

Comment:
I would only hook this if the pin isn't empty

Reply:
It is required in both cases to complete the process to receive the result of bonding.